public/index.html

```diff
@@ -0,0 +1 @@
+<!doctype html><html><head><title>Barefoot Nomad</title></head><body><div id="root"></div><script type="text/javascript" src="/bundle.js"></script><script type="text/javascript" src="/bundle.js"></script><script type="text/javascript" src="/bundle.js"></script></body></html>
```

Comment:
awesome, but I don't understand why you are reapeating this `<script type="text/javascript" src="/bundle.js">` three times